wee_slack.py

```diff
@@ -1152,9 +1152,12 @@ def update_member_presence(self, user, presence):
                 c.update_nicklist(user.id)
 
     def subscribe_users_presence(self):
+        # FIXME: We should try to be smarter to fetch the users whom we
```

Comment:
Can you write that this is done because of a size limitation in the API?

Reply:
✓